nuspec/dotnet-mono.nuspec

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package>
     <metadata>
-        <id>KRE-CLR-x86</id>
+        <id>dotnet-mono</id>
         <version>0.0.1-pre</version>
         <authors>Microsoft</authors>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
-        <description>The K Runtime Environment for Desktop CLR x86</description>
+        <description>The .NET Runtime Environment for Mono on OSX</description>
```

Comment:
Just "for Mono" I think. It's the same package on Linux/BSD/OSX/etc. because we don't have a native host for Mono, we boot straight to managed code.